docs/pages/enroll-resources/database-access/enroll-self-hosted-databases/sql-server-ad-pkinit.mdx

```diff
@@ -219,6 +219,14 @@ database section below as appropriate:
 - `spn`: Service Principal Name (SPN) for SQL Server to fetch Kerberos tickets.
 - `kdc_host_name`: SPN of the domain controller responsible for providing the LDAP CA.
 - `ldap_cert`: The contents of the LDAP CA previously exported.
+- `ldap_service_account_name`: Name of the service account Teleport uses to query LDAP for user SIDs.
+- `ldap_service_account_sid`: SID corresponding to the specified `ldap_service_account_name`.
+
+<Admonition type="warning" title="Important">
```

Comment:
Instead of a warning, what do you think to have a step that walks user through on how to create this service account, what permission it needs, how to grab the SID, etc? Is there anything we can use from the windows guide?

Reply:
I've changed the admonition type, making it more of a note about the (minimal) requirements. I'm pretty sure these permissions are given by default.